Sources/PackageCollections/Providers/GitHubPackageMetadataProvider.swift

```diff
@@ -235,6 +240,23 @@ extension GitHubPackageMetadataProvider {
 }
 
 extension GitHubPackageMetadataProvider {
+    fileprivate struct Release: Codable {
+        let name: String
+        let tagName: String?
+        // This might contain rich-text
+        let body: String?
```

Comment:
is it markdown?

Reply:
Yes it can be - basically whatever is typed into the text input box

Reply:
gotcha :+1: